index.html

```diff
@@ -9,17 +9,53 @@
   <title>Document</title>
 </head>
 <body>
+  <header>
+    <nav>
+      <div>
+        <h3>Awesome Books</h3>
+      </div>
+      <div>
+        <ul class="nav-list">
+          <li class="list-btn"><a class="a-color" href="#book-list">List</a></li>
+          <li class="vector"></li>
+          <li class="add-new-btn"><a class="b-color" href="#add-book-form">Add new</a></li>
+          <li class="vector"></li>
+          <li class="contact-btn"><a class="c-color" href="#contact-info">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <div class="date"></div>
+  </header>
+  
   <main>
-    <h1>All awesome books</h1>
-    <section class="books-container"></section>
-    <div class="vector"></div>
-    <h2>Add a new book</h2>
-    <form action="#" class="form-class">
-      <input id="title" type="text" placeholder="Title">
-      <input id="author" type="text" placeholder="Author">
-      <button id="add-btn">Add</button>
-    </form>
+    <section class="book-list show" id="book-list">
+      <h1>All awesome books</h1>
+      <section class="books-container"></section>
+    </section>
+
+    <section class="add-book-form hide" id="add-book-form">
+      <h2>Add a new book</h2>
+      <form action="#" class="form-class">
+        <input id="title" type="text" placeholder="Title">
+        <input id="author" type="text" placeholder="Author">
+        <button id="add-btn">Add</button>
+      </form>
+    </section>
+
+    <section class="contact-info hide" id="contact-info">
+      <h2>Contact information</h2>
+      <p>Do you have any questions or you just want to say "Hello"?</p>
+      <p>You can reach out us!</p>
+      <ul>
+        <li>Our e-mail: mail@mail.com</li>
+        <li>Our phone number: 0043356534422</li>
+        <li>Our address: Stretname 22, 84503 City, Country</li>
+      </ul>
+    </section>
+    
```

Comment:
- [ ] You forgot to add a footer, kindly add this requirement to your project 😄 
    - Your project:
      ![imagen](https://user-images.githubusercontent.com/60789317/143494813-905658df-f232-407e-90bf-61e1a98d77fd.png)
    - Template:
      ![imagen](https://user-images.githubusercontent.com/60789317/143494921-e0a1496d-054f-4b3a-83d5-66d2bc8b54de.png)